kitchen-tests/kitchen.yml

```diff
@@ -75,6 +75,11 @@ platforms:
     pid_one_command: /sbin/init
     intermediate_instructions:
       - RUN sed -i -e "s/Defaults.*requiretty.*/Defaults    !requiretty/g" /etc/sudoers
+      - RUN wget http://people.centos.org/tru/devtools-2/devtools-2.repo -O /etc/yum.repos.d/devtools-2.repo
+      - RUN yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-gcc-c++
+      - RUN ln -s /opt/rh/devtoolset-2/root/usr/bin/gcc /usr/local/bin/gcc
+      - RUN ln -s /opt/rh/devtoolset-2/root/usr/bin/c++ /usr/local/bin/c++
+      - RUN ln -s /opt/rh/devtoolset-2/root/usr/bin/g++ /usr/local/bin/g++
```

Comment:
what about following https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/ ?

Reply:
That's GCC 7 vs. 4.8. All we needed was 4.8 vs. 4.7, but your method is probably easier to install so I'll pull that in.